components/apimgt/org.wso2.carbon.apimgt.rest.api.gateway.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/gateway/v1/impl/SequenceApiServiceImpl.java

```diff
@@ -27,22 +27,31 @@
 import org.wso2.carbon.apimgt.api.gateway.GatewayContentDTO;
 import org.wso2.carbon.apimgt.gateway.InMemoryAPIDeployer;
 import org.wso2.carbon.apimgt.gateway.utils.SequenceAdminServiceProxy;
+import org.wso2.carbon.apimgt.impl.APIConstants;
 import org.wso2.carbon.apimgt.rest.api.gateway.v1.*;
 import org.apache.cxf.jaxrs.ext.MessageContext;
 import org.wso2.carbon.apimgt.rest.api.util.utils.RestApiUtil;
 
 import javax.ws.rs.core.Response;
+import java.util.Map;
 
 public class SequenceApiServiceImpl implements SequenceApiService {
 
     private static final Log log = LogFactory.getLog(SequenceApiServiceImpl.class);
+    private final String SUPER_TENAT_DOMAIN = "carbon.super";
```

Comment:
Point to the constant in APIConstants.

Reply:
https://github.com/wso2/carbon-apimgt/pull/9009/commits/b99a273017722380eaa25c62f86587c522498b9a